scripts/utils/test_integration-functions.R

```diff
@@ -136,3 +136,25 @@ calculate_kbet(combined_sce, "batch", unintegrated=TRUE, seed = 2022,
 calculate_batch_asw(combined_sce, unintegrated=TRUE)
 calculate_batch_ari(combined_sce, unintegrated=TRUE)
 calculate_lisi(combined_sce, unintegrated=TRUE)
+
+# Test reverse ARI 
+individual_sce_dir <- here::here("data", "scib_simulated", "sce")
+sce_ids <- c(
+  "sim1_Batch1",
+  "sim1_Batch2",
+  "sim1_Batch3",
+  "sim1_Batch4",
+  "sim1_Batch5",
+  "sim1_Batch6"
+)
+ind_sce_files <- glue::glue("{individual_sce_dir}/{sce_ids}_sce.rds")
```

Comment:
check that my spacing doesn't end up wonky please, suggestions spacing is always eh... 

```suggestion
ind_sce_files <- file.path(
  individual_sce_dir, 
  glue::glue("{sce_ids}_sce.rds")
)
```